### PR TITLE
Feature/buildfix

### PR DESCRIPTION
**Pull request by nsoblath**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -113,22 +125,26 @@ else( Dripline_BUILD_PYTHON )
...
 
-add_definitions( -DDL_MAX_PAYLOAD_SIZE=${Dripline_MAX_PAYLOAD_SIZE} )
+#add_library( ${PROJECT_NAME}_void INTERFACE )
+#export( TARGETS ${PROJECT_NAME}_void
+#    FILE ${PROJECT_BINARY_DIR}/${PROJECT_NAME}Targets.cmake
+#)
+#install( TARGETS ${PROJECT_NAME}_void 
+#    EXPORT ${PROJECT_NAME}Targets 
+#    LIBRARY DESTINATION ${LIB_INSTALL_DIR} 
+#)
```
Comment (laroque) [inline CMakeLists.txt:144]: is this coming back? maybe go ahead and remove?

**Comment by laroque** [comment]
There were a few smaller blocks of commented out lines also. I think if we're confident that it works now we should probably drop them (unless we're expecting them to come back in with some future feature). Otherwise it LGTM so I'll approve.

File: CMakeLists.txt (hunk 1 above)
Comment (nsoblath) [reply CMakeLists.txt]: Done

**Comment by nsoblath** [comment]
I removed the extra lines

**Pull request closed without merging**